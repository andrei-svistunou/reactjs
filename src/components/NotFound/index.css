.h2 {
  font-size: 40px;
  font-weight: 400;
  text-align: center;
  margin-top: 40px;
}