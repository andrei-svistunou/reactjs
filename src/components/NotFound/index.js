import React from 'react';
import PropTypes from 'prop-types';
import './index.css';

const NotFound = props => (
  <h2 className="h2">{props.text}</h2>
);

NotFound.propTypes = {
  text: PropTypes.string
};

NotFound.defaultProps = {
  text: 'No films found'
};

export default NotFound;
