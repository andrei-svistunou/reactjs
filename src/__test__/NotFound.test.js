import React from 'react';
import { shallow } from 'enzyme';
import toJson from 'enzyme-to-json';
import NotFound from '../components/NotFound';

describe('<NotFound />', () => {
  test('should render <Label /> with props', () => {
    const notFound = shallow(<NotFound text='NotFound text' />);

    expect(notFound.text()).toEqual('NotFound text');
    expect(toJson(notFound)).toMatchSnapshot();
  });
});
