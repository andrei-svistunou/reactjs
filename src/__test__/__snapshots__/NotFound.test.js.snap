// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<NotFound /> should render <Label /> with props 1`] = `
<h2
  className="h2"
>
  NotFound text
</h2>
`;
